lib/sender.js

```diff
@@ -135,6 +135,10 @@ class Sender {
 
     if (skipMasking) return [target, data];
 
+    if (typeof data === 'string') {
+      data = Buffer.from(data);
+    }
```

Comment:
This isn't strictly need but makes things safer for bugs introduced by future changes.

Reply:
Not very happy with this. I would prefer to use `data.length` below.